hyperspy/misc/axis_tools.py

```diff
@@ -20,7 +20,7 @@
 
 
 def check_axes_calibration(ax1, ax2):
-    """Check if the calibration of two DataAxis objects matches.
+    """Check if the calibration of two Axis objects matches.
```

Comment:
This function is used only once in hyperspy, would it be more simple to remove this function and instead use `np.allclose(ax1.axis, ax2.axis)`? This approach would be more transparent and would have the advantage of specifying the tolerance, when necessary.

Reply:
The tolerance could be added here, but it is true that as all three axes types do have an axis array as attribute, the simple comparison of these may be sufficient, but probably still checking units afterwards as well?

Reply:
`np.allclose` fails on axes of different size that are compared, so I think having this function still makes sense, but I have simplified it extremely by comparing the axes vectors.